.circleci/config.yml

```diff
@@ -42,9 +42,13 @@ jobs:
             pip3 install --no-cache-dir -r dev-requirements.txt
           name: Install dependencies
 
+      # Ideally we'd use standard docker build since we're building for the native
+      # platform. Unfortunately the singleuser image can't be built on arm64, and the
+      # easiest way to skip this image is to user chartpress's --platform option,
+      # which is only supported with docker-buildx engine
       - run:
           command: |
-            chartpress
+            chartpress -builder docker-buildx
```

Comment:
`--builder`, right?

Reply:
Yes, along with needing to be explicit about the platform.